nomad/traefik-wall/deploy/traefik.tpl.yml

```diff
@@ -10,6 +10,7 @@ entryPoints:
       trustedIPs:
         - 128.112.200.245/32
         - 128.112.201.34/32
+        - 10.0.0.0/8
```

Comment:
wanna just add these then?

https://github.com/pulibrary/princeton_ansible/blob/main/group_vars/all/vars.yml#L141

Reply:
Oh is https://www.net.princeton.edu/ip-network-ranges.html not right? That's what I have in exemptIps

Reply:
It turns out I didn't need this, just previous exemptIps, and upgrading the plugin.

Reply:
The network ranges you have a correct for your use case. The group vars are "the library segment" of that range.